csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair.html

```diff
@@ -0,0 +1,11 @@
+{% extends "csss/header.html" %}
+{% block content %}
+{{ block.super }}
+	<h2 class="title is-size-2">Computing Science Student Society Tech Fair 2022</h2>

```

Comment:
I am a bit confused why bother having this page? why not just link directly to `csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html`?

Reply:
There will be an additional page added later, this will be a landing page that splits off between both of them. One will be for students, the other for companies. I am adding it now to make the next PR easier.

Reply:
ah